roles/openshift_logging_elasticsearch/tasks/migrate_cluster.yml

```diff
@@ -0,0 +1,25 @@
+---
+- set_fact:
+    es_component: "{{ 'es' ~ ( (openshift_logging_elasticsearch_ops_deployment | default(false) | bool) | ternary('-ops', '') ) }}"
+    es_fact_name: "{{ 'elasticsearch' ~ ( (openshift_logging_elasticsearch_ops_deployment | default(false) | bool) | ternary('_ops', '') ) }}"
+    es_config_name: "{{ 'elasticsearch' ~ ( (openshift_logging_elasticsearch_ops_deployment | default(false) | bool) | ternary('-ops', '') ) }}"
+
+# TODO: be able to also expand the number of cluster members here
+
+- template:
+    src: operator_adopt.j2
+    dest: "{{ tempdir }}/operator_adopt.yml"
+  vars:
+    es_cluster_nodes: "{{ openshift_logging_facts[es_fact_name]['deploymentconfigs'].values() | list }}"
+    es_additional_nodes: "{{ openshift_logging_elasticsearch_replica_count | int - openshift_logging_facts[es_fact_name]['deploymentconfigs'].keys() | count }}"
+    es_cluster_name: "logging-{{ es_component }}"
+    es_cpu_limit: "{{ openshift_logging_elasticsearch_cpu_limit | default('') }}"
+    es_cpu_request: "{{ openshift_logging_elasticsearch_cpu_request | min_cpu(openshift_logging_elasticsearch_cpu_limit | default(none)) }}"
+    es_memory_limit: "{{ openshift_logging_elasticsearch_memory_limit }}"
+    es_node_selector: "{{ openshift_logging_elasticsearch_nodeselector | default({}) }}"
+    pvc_name: "{{ openshift_logging_elasticsearch_pvc_name }}"
+    pvc_size: "{{ (openshift_logging_elasticsearch_pvc_size | trim | length == 0) | ternary('10Gi', openshift_logging_elasticsearch_pvc_size) }}"
+    pvc_access_modes: "{{ openshift_logging_elasticsearch_pvc_access_modes | list }}"
+    pvc_storage_class_name: "{{ openshift_logging_elasticsearch_pvc_storage_class_name | default('', true) }}"
+    es_secret_name: "logging-{{ es_config_name }}"
```

Comment:
As I understand the certificates are the same for both clusters, so it probably makes sense to point to same secret name `logging-elasticsearch` from both `logging-es` and `logging-es-ops` clusters